packages/pwa/app/utils/url.js

```diff
@@ -197,64 +219,84 @@ export const removeQueryParamsFromPath = (path, keys) => {
 }
 
 /**
- * Rebuild the path with locale/site value to the path as url path or url query param
- * based on url config
- * @param {string} url - based url of the output url
+ * Rebuild the path with locale/site values with a given url
+ * The position of those values will based on the url config of current configuration.
+ *
+ * @param {string} relativeUrl - the base relative Url to be reconstructed on
  * @param {object} configValues - object that contains values of url param config
- * @return {string} - an output url
+ * @param {object} opts - options
+ * @return {string} - an output path that has locale and site
  *
  * @example
- * //pwa-kit-config.json
+ * //config/default.json
  * url {
- *    locale: "query_param"
+ *    locale: "query_param",
+ *    site: "path"
  * }
- * buildPathWithUrlConfig('/women/dresses', {locale: 'en-GB'})
  *
- * Returns
- *  /women/dresses?locale=en-GB
- *
- *  @example
- * //pwa-kit-config.json
- * url {
- *    locale: "path"
- * }
- * buildPathWithUrlConfig('/women/dresses', {locale: 'en-GB'})
- *
- * Returns
- *  /en-GB/women/dresses
+ * buildPathWithUrlConfig('/women/dresses', {locale: 'en-GB', site: 'global'})
+ * => /global/women/dresses?locale=en-GB
  *
  */
-export const buildPathWithUrlConfig = (url, configValues = {}) => {
+export const buildPathWithUrlConfig = (relativeUrl, configValues = {}, opts = {}) => {
```

Comment:
```suggestion
export const buildPathWithUrlConfig = (relativeUrl, urlConfigValues = {}, opts = {}) => {
```

I think since both of the function parameters deal with some kind of config (`configValues` are actually for the url config, while `opts.site` is for the current site configuration), it's better if we be more specific and rename `configValues`.